tests/sensors/test_wait_pool.py

```diff
@@ -195,9 +226,13 @@ def test_wait_closed_pool(requests_mock, dag_for_test_wait_closed_pool, pool_map
     conn_uri = conn.get_uri()
 
     with mock.patch.dict('os.environ', AIRFLOW_CONN_TOLOKA_CONN=conn_uri):
+        def task_suites(request, context):
+            return success_answer_map
+
         def pool(request, context):
             return pool_map_with_readonly
```

Comment:
It seems like patching `os.environment` does not affect function definitions. Do we need to patch environment here?